pkgpanda/build/src_fetchers.py

```diff
@@ -310,7 +319,10 @@ def checkout_to(self, directory):
 
         if self.sha != file_sha:
             corrupt_filename = self.cache_filename + '.corrupt'
-            check_call(['mv', self.cache_filename, corrupt_filename])
+            if is_windows:
+                os.replace(self.cache_filename, corrupt_filename)
+            else:
+                check_call(['mv', self.cache_filename, corrupt_filename])
```

Comment:
We should be able to use `os.replace()` on all platforms here.

Reply:
windows filesystem does work differently that Linux. I did not want to do this without first talking to you. It seems that you are OK with it so we can change this.

Reply:
@paulcallen check if this has been updated